src/aac/ui/menu_bar.py

```diff
@@ -0,0 +1,65 @@
+from tkinter import Menu, Tk
```

Comment:
General recommendation:

Disable items that aren't implemented - i.e. that `donothing` - so they can be clearly distinguished as "this doesn't work (yet)".